.github/workflows/verify-api-model.yaml

```diff
@@ -0,0 +1,23 @@
+name: verify api-model
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build-project:
+    runs-on: ubuntu-18.04
+    if: github.repository_owner == 'Apicurio'
+    steps:
+      - name: Set up JDK 11
+        uses: AdoptOpenJDK/install-jdk@v1
+        with:
+          version: 11
+          architecture: x64
+
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
```

Comment:
api-model is an embedded project in the operator repo and we would like to build the project only when there are changes in the api-model project. current implementation will build the project every time there is a push in master. Please add a filter/condition to avoid doing unnecessary builds.

Please refer the below links for reference:-
https://github.com/Apicurio/apicurio-registry/blob/master/.github/workflows/verify.yaml#L179
https://github.com/dorny/paths-filter